compair/tests/api/test_comparisons.py

```diff
@@ -314,12 +322,12 @@ def test_get_and_submit_comparison(self, mocked_update_assignment_grades_run, mo
                     actual_comparison = rv.json['comparison']
                     self._validate_comparison_submit(comparison_submit, actual_comparison, expected_comparison)
 
-                    # grades should increase for every comparison
+                    # grades should not drop after each round of comparison
                     if user.id == self.data.get_authorized_student().id:
                         new_course_grade = CourseGrade.get_user_course_grade(self.course, user)
                         new_assignment_grade = AssignmentGrade.get_user_assignment_grade(self.assignment, user)
-                        self.assertGreater(new_course_grade.grade, course_grade)
-                        self.assertGreater(new_assignment_grade.grade, assignment_grade)
+                        self.assertGreaterEqual(new_course_grade.grade, course_grade)
```

Comment:
this is incorrect, grade should always increase

Reply:
If the student do more comparisons than required (which normally won't happen via GUI), the score won't increase.  The test here is explicitly trying to do all available comparisons and trigger error 400.

Will change the checking to:
- if doing comparisons required by the assignment, grade should increase;
- otherwise the grade should stay the same

Reply:
Thanks, sounds great